### DISCUSSION TITLE
Include Sidekiq wiki content in Google search results

### DISCUSSION THREAD

**Discussion by adamlogic** [sidekiq/sidekiq#6109] [Ideas]
It's a bummer that all the helpful content in the wiki is [hidden from Google search](https://www.google.com/search?client=safari&rls=en&q=site%3Ahttps%3A%2F%2Fgithub.com%2Fsidekiq%2Fsidekiq%2Fwiki%2F&ie=UTF-8&oe=UTF-8). This is [intentional on the part of GitHub](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis):

> Note: Search engines will only index wikis with 500 or more stars that you configure to prevent public editing. For more information, see "[Changing access permissions for wikis](https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis)."
> 
> If you need search engines to index your content, you can use [GitHub Pages](https://docs.github.com/en/pages) in a public repository.

So the only way for wiki content to be searchable (not counting GitHub search) is to prevent public editing. This would mean more work for Mike, or maybe he could designate people with wiki access (I'm not sure).

Those are significant trade-offs, but I wanted to bring it up because there's so much helpful content in the wiki that many people never find because they're using Google, not GitHub search.

**Comment by mperham** [comment]
Oh wow, I didn't know about that restriction. I've been frustrated with Sidekiq's poor documentation SEO for some time, thanks for educating me. I think Sidekiq might be at the stage where turning off public editing is ok. I can probably keep up with the flow of documentation changes on my own without being overwhelmed.

If anyone wants to be on the "documentation team", I can provide commit access so you can still edit.

**Reply by adamlogic** [reply]
✋ I'm happy to help!

**Reply by JuanitoFatas** [reply]
Happy to help 👋🏻

**Reply by JuanitoFatas** [reply]
Thanks for the invite @mperham, but I still cant edit the wiki after accepted invitation 🤔

**Reply by mperham** [reply]
Weird, you have to have push access to get wiki edit rights. Fixed.
